tests/basic/sharedv4_svc_test.go

```diff
@@ -272,6 +272,27 @@ func (fm *failoverMethodReboot) sleepBetweenFailovers() time.Duration {
 	return 2 * time.Minute
 }
 
+type failoverMethodRecover struct {
```

Comment:
The term "Recover" does not make sense in this context to me at least. That term was used in the context of recovering the node which is in StorageDown state. We are not doing that here. We should probably call  it failoverMethodMaintenance instead.

Reply:
updated